wiki/plugins/metadata/static/wiki/css/metadata.css

```diff
@@ -13,7 +13,7 @@ ul.inline,ul.inline > li { display: inline-block; }
 ul.inline { padding: 0; }
 ul.inline > li { margin: 0px 5px; }
 
-.wiki-article { font-family: 'Roboto', sans-serif; }
+.wiki-article { font-family: 'Roboto', sans-serif; font-size: 14pt; }
```

Comment:
Will this affect Construal, Usage, and Adposition articles?

Reply:
Yep, all articles. I figure it's easier to read the larger font.